index.html

```diff
@@ -33,7 +33,7 @@
               </ul>
             </nav>
             <div class="introduce-myself"> 
-              <img class="my-photo" src="./img/46458592.jfif" height="148px" width="148px" alt="it is i))))">
+              <img class="my-photo" src="./img/nazar.jpg" height="148px" width="148px" alt="it is i))))">
```

Comment:
change the alt
`it is i` -> `it's me` or `me`... something like that

And also need to change the image photo name 
`nazar.jpg` -> `me.jpg` or something like that

Reply:
ok)